<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人文互联网-高效办公工具分享</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_filepath"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="open_filepath">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>选择文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>合并工作表</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushbutton_start">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(7, 193, 96);
font: 11pt &quot;Agency FB&quot;;
font: 11pt &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>开始合并</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_openfolder">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(7, 193, 96);
font: 11pt &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string>打开结果文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;黑体&quot;;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600;&quot;&gt;左手excel,右手python，让你繁琐的工作自动化！&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600;&quot;&gt;欢迎关注我的公众号：人文互联网！&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600;&quot;&gt;回复关键词&lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;&amp;quot;高效办公&amp;quot; &lt;/span&gt;&lt;span style=&quot; font-family:'SimSun'; font-weight:600; color:#000000;&quot;&gt;即可获取下载链接。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/qrcode/wechat_search.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/imgcode.qrc"/>
 </resources>
 <connections/>
</ui>
